spec/acceptance/rails_middleware_spec.rb

```diff
@@ -16,5 +16,17 @@
       @app.initialize!
       assert @app.middleware.include?(Rack::Attack)
     end
+
+    it "can be configured via a block" do
```

Comment:
the PR goal (per its title) seems to be supporting the ability to have multiples instances of Rack::Attack. I'm wondering.. should we have a test that demonstrates how that works? Not only for catching regressions but also to document this new "feature". Thoughts?